cci-config-generator.sh

```diff
@@ -9,13 +9,14 @@ echo -e "Gathering info please wait..."
 circle_dir="./.circleci"
 conf_file="./.circleci/config.yml"
 vcs_provider="$(git remote -v | awk -F'[@.]' 'NR==1{ print $2 }')"
```

Comment:
$ __git remote -v__
```
origin	https://github.com/cclauss/ckan (fetch)
origin	https://github.com/cclauss/ckan (push)
upstream	https://github.com/ckan/ckan (fetch)
upstream	https://github.com/ckan/ckan (push)
```
$ __echo $(git remote -v | awk -F'[@.]' 'NR==1{ print $2 }')__
```
com/cclauss/ckan (fetch)
```

Reply:
AH interesting, I never looked at the VCS detection, just project name.  I'll push something up for this too.  THanks!